### DISCUSSION TITLE
Order All of results by newest result (orderby ('id' 'DESC'))

### DISCUSSION THREAD

**Discussion by MehranLabour** [yajra/laravel-datatables#2860] [Q&A]
i want when any of columns sorted the result sorted by newest result, by default its sorted by oldest results.

default sort of result after sorting by name is like below 

```
|id| |name| |Address| 

|2| |Alex|   |Lorem ipsum|

|3| |Alex|    |Lorem ipsum|

|5| |Alex|    |Lorem ipsum|

|1| |James| |Lorem ipsum|

|4| |Sara|    |Lorem ipsum|

```

 Butt I want Results Like below
like below after sort the results by name i want the same names sort by newest in top

```
|id| |name| |Address| 

|5| |Alex|   |Lorem ipsum|

|3| |Alex|    |Lorem ipsum|

|2| |Alex|    |Lorem ipsum|

|1| |James| |Lorem ipsum|

|4| |Sara|    |Lorem ipsum|

```

i hope you understand my problem.

- Operating System:
- PHP Version: 7.1.3
- Laravel Version:5.8.*
- Laravel-Datatables Version  8.0

**Comment by yajra** [comment]
On js, set something like `order: [[1, 'desc']]` where 1 is the index of the column you want to be sorted on initial load.

**Comment by MehranLabour** [comment]
Thanks,

**Comment by jbnv** [comment]
What if the column for the initial sort is not one that's displayed?

**Comment by yajra** [comment]
@jbnv use `data` for display and `name` for search/sort functionality.

**Comment by jbnv** [comment]
@yajra I don't understand this response. I removed the `aaSorting` property from the `$("#table").DataTable()` argument. Yet the table still sorts by the first column with no other input. I don't want to do this. I want the table to use a default ordering that I specify in the controller *unless* someone clicks to sort one of the columns.
